doc/fluxing.rst

```diff
@@ -72,6 +72,28 @@ The algorithm options are:
  - IR   = Should be used for data with lambbda > 7000A.
    Peforms joint fit for sensitivity function and telluric absorption using HITRAN models.
 
+--sens
+++++++
+
+Provide a file to guide the process.  Do this if your changes to
+the defaults are not accommodated by the script inputs.
+
+IR without a Standard
+---------------------
+
+If you wish to generate a sensitivity function on a standard
+start that is not part of the PypeIt database and are working
+in the IR, you can feed the stellar parameters.  Here is an
```

Comment:
Please add one sentences about where the model comes from. 
For A0, we will use VEGA spectrum.
For other type of stars, we will use Kurucz93 stellar SED.

Reply:
done